build.sbt

```diff
@@ -345,13 +351,7 @@ lazy val core = (projectMatrix in file("core"))
     scalaVersions = List(scala2_11),
     settings = {
       commonNativeSettings ++ intellijSkipImport ++ List(
-        libraryDependencies ++= Seq(
-          "org.scala-native" %%% "test-interface" % scalaNativeTestInterfaceVersion % Test,
```

Comment:
so we don't need the test-interface anymore?

Reply:
It is automatically added to the `Test` scope by the Scala Native sbt plugin.
Verified by running `show upickleNative2_11/Test/libraryDependencies` which lists it.